client/src/components/App.js

```diff
@@ -37,7 +37,7 @@ const App = (props) => {
         <AuthenticatedRoute exact path="/home" component={HomePage} user={currentUser} />
         <AuthenticatedRoute exact path="/ingredients" component={IngredientsList} user={currentUser} />
         <AuthenticatedRoute exact path="/ingredients/:id" component={IngredientShow} user={currentUser} />
-        <AuthenticatedRoute exact path="/addIngredient" component={NewIngredient} user={currentUser} />
+        <AuthenticatedRoute exact path="/add_ingredient" component={NewIngredient} user={currentUser} />
```

Comment:
oh my god I totally mispoke IM SO SORRY

I didnt mean snake case, I just meant dashes between lowercase words `/add-ingredient`

Even more conventional would be `/ingredients/new`